### ISSUE TITLE
Show number of rules for each filter

### ISSUE THREAD

**Issue by pietrodn** [AdguardTeam/AdguardForiOS#113]
Hi, it would be useful to see in the filter properties the number of rules of each filter. In this way, if the user hits the maximum number of rules, they know which ones to remove.

**Comment by ameshkov** [comment]
The problem with this is that we don't know how many rules will be left after filters compilation process.

**Example**

Filter1:
```
###banner1
###banner2
```

Filter2:
```
###banner3
###banner4
```

Resulting content blocker will have only one rule:
```
[
    {
        "action":
        {
            "type": "css-display-none",
            "selector": "#banner1, #banner2, #banner3, #banner4"
        }
    }
]
```

Maybe instead of the in-app filters compilation we should do it on the server side. This will make this all much easier. @Stillness-2 what do you think about it?

**Comment by ameshkov** [comment]
available in v3.0 (content blockers screen)
